service/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -29,3 +29,6 @@ databaseChangeLog:
   - include:
       file: changesets/20220412_add_last_encrypt_timestamp_to_ssh_key_pair_table.yaml
       relativeToChangelogFile: true
+  - include:
```

Comment:
There's a merge conflict here. Make sure this changeset comes _after_ the just-merged changeset.